Meadowlark-js/tests/e2e/scenarios/QueryStringValidation.test.ts

```diff
@@ -14,7 +14,7 @@ describe('When retrieving information', () => {
       await baseURLRequest()
         .get('/v3.3b/ed-fi/schools')
         .auth(await getAccessToken('vendor'), { type: 'bearer' })
-        .expect(200)
+        .expect(502)
```

Comment:
200 is the correct response here, not 502. If we're getting a 502 now, then that means there is a communication problem and the test is not working as expected. For GitHub Actions, we might need to extend the timeout to longer than 10 seconds.

Reply:
https://github.com/Ed-Fi-Exchange-OSS/Meadowlark/actions/runs/4120175386/jobs/7114667292

> "2023-02-08 02:30:41.923 error 985ca835-d44a-4160-9529-feba81045f75 opensearch.repository.QueryOpensearch.queryDocuments OpenSearch Error: index not found (ResponseError) - Error occurred in OpenSearch engine: no such index [ed-fi$3-3-1-b$school]"

I get it now: we always had the error, because there is _nothing_ in OpenSearch, and therefore no index yet. This is an interesting scenario. Looks like we need to continue fine-tuning the response handling. I'm thinking about that.